cms/search/publishers.py

```diff
@@ -132,6 +132,7 @@ def __init__(self) -> None:
             bootstrap_servers=[settings.KAFKA_SERVER],
             api_version=settings.KAFKA_API_VERSION,
             value_serializer=lambda v: json.dumps(v).encode("utf-8"),
+            retries=5,
```

Comment:
Let's making this a setting with a default.

Reply:
We are leaving this as is =)